proxy/delete.go

```diff
@@ -16,18 +16,24 @@ import (
 
 // DeleteFunction delete a function from the OpenFaaS server
 func DeleteFunction(gateway string, functionName string, tlsInsecure bool) error {
-	return DeleteFunctionToken(gateway, functionName, tlsInsecure, "")
+	return DeleteFunctionToken(gateway, functionName, tlsInsecure, "", "")
```

Comment:
why is the namespace empty here? It seems like both `DeleteFunction` and `DeleteFunctionToken` should accept `namespace`

Reply:
Done :+1: